components/Tag/index.js

```diff
@@ -0,0 +1,69 @@
+import Link from "next/link";
+import Image from "next/image";
+import styled, { ThemeProvider } from "styled-components";
+
+export default function Tag({tagId, headline, subHeadline, image, tagType }) {
+  
+    return (
+      <Link href={`/${tagType}/${tagId}`}>
+        <ThemeProvider theme={tagType === "plants" ? plantTagtheme : tipTagtheme}>
+        <StyledTag>
+       
+       <StyledImage src={image} width={55} height={55} alt={`"image of ${headline}"`}/>
```

Comment:
You can skip the quotes, otherwise your alt attribute will look likes this: `alt=""image of plant""`